[Fix #495] Default rails console strategy configuration (#780)
toptal/chewy: CHANGELOG.md

### Changes

  * [#780](https://github.com/toptal/chewy/pull/780): changing default console strategy ([@Vitalina-Vakulchyk][]):
    * ability to set default console strategy with `Chewy.console_strategy`

>     * `:urgent` is used by default in case, where `Chewy.console_strategy` is not set

Reviewer: ```suggestion
    * `:urgent` is used by default if `Chewy.console_strategy` is not set
```

>     * `:urgent` is used by default in case, where `Chewy.console_strategy` is not setted

Reviewer: ```suggestion
    * `:urgent` is used by default if `Chewy.console_strategy` is not setted
```

### Changes

>   * [#780](https://github.com/toptal/chewy/pull/780): Changing default console strategy ([@Vitalina-Vakulchyk][]):
>     * ability to set default console strategy with `Chewy.console_strategy`
>     * `:urgent` is used by default if `Chewy.console_strategy` is not set

Reviewer 2: ```suggestion
  * [#495](https://github.com/toptal/chewy/issues/495): Ability to change Rails console strategy with `Chewy.console_strategy` ([@Vitalina-Vakulchyk][]):
```

### Changes

>   * [#780](https://github.com/toptal/chewy/pull/780): Changing default console strategy ([@Vitalina-Vakulchyk][]):
>     * ability to set default console strategy with `Chewy.console_strategy`
>     * `:urgent` is used by default if `Chewy.console_strategy` is not set
> 
>   * [#778](https://github.com/toptal/chewy/pull/778): **(Breaking)** Drop support for Ruby 2.5 ([@Vitalina-Vakulchyk][])
> 

Reviewer 2: ```suggestion
  * [#780](https://github.com/toptal/chewy/pull/780): Changing default console strategy ([@Vitalina-Vakulchyk][]):
    * ability to set default console strategy with `Chewy.console_strategy`
    * `:urgent` is used by default if `Chewy.console_strategy` is not set
  * [#778](https://github.com/toptal/chewy/pull/778): **(Breaking)** Drop support for Ruby 2.5 ([@Vitalina-Vakulchyk][])
```
